modules/local/add_most_severe_consequence.nf

```diff
@@ -0,0 +1,43 @@
+process ADD_MOST_SEVERE_CSQ {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "conda-forge::python=3.9.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'quay.io/biocontainers/python:3.9--1' }"
```

Comment:
The versions between conda and the containers are not matching

Reply:
Fixed 👍🏻